crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -81,17 +80,10 @@ impl JsonRpcHandler {
         block_id: BlockId,
         contract_address: ContractAddressHex,
     ) -> RpcResult<ClassHashHex> {
-        let parsed_address = contract_address.0.try_into().map_err(|_| {
-            error::ApiError::RpcError(RpcError::invalid_params(format!(
-                "Invalid contract_address: {:?}",
```

Comment:
Just an observation that this way we are losing the contract address from the error message, right?

Reply:
Now not